.github/workflows/hza-data-notebook.yml

```diff
@@ -0,0 +1,39 @@
+name: Run notebook file
+
+on:
+  push:
+    branches:
+    - main
+
```

Comment:
Can you add a `workflow_dispatch: {}` so we can call this action manually?

Reply:
Is this instead of the push on main?